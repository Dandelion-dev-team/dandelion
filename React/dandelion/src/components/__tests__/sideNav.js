import React from "react"
import renderer from "react-test-renderer"
import SideNav from "../sideNav"

describe("Project Card", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SideNav/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})