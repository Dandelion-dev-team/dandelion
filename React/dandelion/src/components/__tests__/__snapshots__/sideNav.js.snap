// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Card renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="sidenav"
  >
    <nav>
      <div
        className="logo"
      >
        <a
          exact={true}
          href="/maintenance/superuser/superuser-dashboard"
        >
          <img
            src="test-file-stub"
          />
        </a>
      </div>
      <a
        href="/maintenance/superuser/superuser-dashboard"
      >
        Dashboard
      </a>
      <a
        href="/maintenance/superuser/project-maintenance"
      >
        Projects
      </a>
      <a
        href="/maintenance/superuser/superuser-help"
      >
        Help
      </a>
      <a
        href="/maintenance/superuser/superuser-alerts"
      >
        Alerts
      </a>
      <a
        href="/maintenance/superuser/school-user-maintenance"
      >
        Students
      </a>
      <a
        href="/maintenance/superuser/superuser-templates"
      >
        Templates
      </a>
      <a
        href="/maintenance/superuser/superuser-settings"
      >
        Settings
      </a>
      <div
        className="news-pane"
      >
        <h3>
          News
        </h3>
      </div>
      <div
        className="btn-row"
      >
        <button
          className="button"
          onClick={[Function]}
        >
          Log Out
        </button>
      </div>
    </nav>
  </div>
</div>
`;
